test/controllers/api/v1/stake_accounts_controller_test.rb

```diff
@@ -24,12 +24,14 @@ def setup
 
   test 'request should return correct results' do
     create(:stake_account, network: 'testnet')
+    create(:stake_pool, network: 'testnet')
     get api_v1_stake_accounts_index_url(network: 'testnet'), headers: { 'Token' => @user.api_token }
     assert_response 200
 
     assert_equal 1, response_to_json(@response.body)['stake_accounts'].size
     assert_equal 'testnet', response_to_json(@response.body)['stake_accounts'].first['network']
     assert_equal 1, response_to_json(@response.body)['total_count']
     assert_equal 200, response_to_json(@response.body)['total_stake']
+    assert_equal 1, response_to_json(@response.body)['stake_pools'].size
```

Comment:
You could also check if correct fields are returned from api for one record. Might be useful when sql or json fields will be changed in future.